pilot/pilot_compute_service.py

```diff
@@ -212,13 +206,3 @@ def __repr__(self):
     def custom_method(self):
         # Custom logic or functionality
         pass
-
-
-def dask_submit(dask_pilot):
```

Comment:
Removed this for now. we can also have PilotComputeService.task , where a task can be executed on a group of pilots.

Reply:
and this i can add it in future, when we deal with multiple pilots.